db/seeds.rb

```diff
@@ -1,23 +1,23 @@
 
 item_contents = ([{ name: "Chocolate Toast", description: "Dark chocolate on toast",
-        price: 3.00, retired: true}, { name: "Cake Toast", description: "Lemon poppyseed cake on toast", price: 4.00},
-        { name: "Brownie Toast", description: "Delicious dark chocolate brownie on toast", price: 4.00},
+        price: "3.00", retired: true}, { name: "Cake Toast", description: "Lemon poppyseed cake on toast", price: "4.00"},
+        { name: "Brownie Toast", description: "Delicious dark chocolate brownie on toast", price: "4.00"},
         { name: "Cinnamon Sugar Toast", description: "Toast sprinkled with raw cane sugar, fresh ground
-        cinnamon, and a hint of sea salt", price: 2.50}, { name: "Avocado Toast", description: "Fresh
-        California avacado sliced over toast", price: 5.00}, { name: "Egg Toast", description: "Choose
-        from fried or scrambled eggs!", price: 4.00}, { name: "Dry Toast", description: "Nothing on it",
-        price: 2.00}, { name: "Tuna Salad Toast", description: "With pickles,onions, and a hint of basil",
-        price: 4.50}, { name: "Buttered Toast", description: "Creamy melted butter", price:
-        5.00}, { name: "Mac and Cheese Toast", description: "Rich, three cheese mac and cheese, on toast",
-        price: 4.50}, { name: "Salad Toast", description: "House salad on toast", price: 4.00}, { name:
-        "Peanut Butter Toast", description: "Fresh ground peanut butter", price: 7.00}, { name:
-        "Sausage Toast", description: "German style brat on toast", price: 6.00}, { name: "Jam Toast",
-        description: "Strawberry, or blueberry", price: 5.50}, { name: "Chicken Toast", description: "Fried
-        or grilled!", price: 8.00}, { name: "Eggplant Toast", description: "Roasted Eggplant on toast",
-        price: 5.00}, { name: "Curry Toast", description: "Spicy and delicious", price: 8.00}, { name:
-        "Steak Toast", description: "Flank Steak", price: 7.50}, { name: "Tofu Toast", description:
-        "Marinated tofu", price: 4.00}, { name: "Salmon Toast", description: "Alaskan made!", price:
-        8.00}, { name: "Cheese Toast", description: "melty sharp cheddar", price: 3.00}])
+        cinnamon, and a hint of sea salt", price: "2.50"}, { name: "Avocado Toast", description: "Fresh
+        California avacado sliced over toast", price: "5.00"}, { name: "Egg Toast", description: "Choose
+        from fried or scrambled eggs!", price: "4.00"}, { name: "Dry Toast", description: "Nothing on it",
+        price: "2.00"}, { name: "Tuna Salad Toast", description: "With pickles,onions, and a hint of basil",
```

Comment:
Line is too long. [108/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.